beacon-chain/core/blocks/block_operations.go

```diff
@@ -808,6 +834,13 @@ func ProcessVolundaryExits(
 ) (*pb.BeaconState, error) {
 	var err error
 	exits := body.VoluntaryExits
+	if uint64(len(exits)) > params.BeaconConfig().MaxVoluntaryExits {
```

Comment:
we should short cut all these operation checks at the beginning of `process_operation` `https://github.com/prysmaticlabs/prysm/blob/spec-v0.6/beacon-chain/core/state/transition.go#L257`

so it doesn't get to the last process operation step and then realize it's a invalid block, it's wasteful